packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -990,12 +986,12 @@ class _TableState<T> extends State<_Table<T>> with AutoDisposeMixin {
                   child: Focus(
                     autofocus: true,
                     onKey: (_, event) => widget.handleKeyEvent != null
-                        ? widget.handleKeyEvent(
+                        ? widget.handleKeyEvent!(
                             event,
                             scrollController,
                             constraints,
                           )
-                        : false,
+                        : false as KeyEventResult,
```

Comment:
this seems wrong will likely throw a cast exception. We should assign to another value of KeyEventResult. Perhaps KeyEventResult.ignored

Reply:
done